$NetBSD: patch-mac_CMakeLists.txt,v 1.4 2025/06/30 22:51:46 gdt Exp $

Make sure QCATOOL_EXECUTABLE is available to 1qt.cmake
This is in conjunction with patch-mac_cmake_1qt.cmake.in

Not yet reported upstream.

--- mac/CMakeLists.txt.orig	2025-05-16 12:02:00.000000000 +0000
+++ mac/CMakeLists.txt
@@ -1,6 +1,6 @@
 # mac bundling must happen at end, so all binaries installed
 # and install_names can be adjusted
-if (APPLE AND QGIS_MACAPP_BUNDLE GREATER -1)
+if (XAPPLE AND QGIS_MACAPP_BUNDLE GREATER -1)
   # for included scripts that set policies (ie OS X bundling)
   install (CODE "cmake_policy(SET CMP0011 NEW)")
   configure_file (cmake/0vars.cmake.in 0vars.cmake @ONLY)
@@ -8,6 +8,7 @@ if (APPLE AND QGIS_MACAPP_BUNDLE GREATER
   install (SCRIPT ${CMAKE_BINARY_DIR}/mac/0qgis.cmake)
   if (QGIS_MACAPP_BUNDLE GREATER 0)
     # start with Qt
+    FIND_QCATOOL(1)
     configure_file (cmake/1qt.cmake.in 1qt.cmake @ONLY)
     install (SCRIPT ${CMAKE_BINARY_DIR}/mac/1qt.cmake)
     if (QGIS_MACAPP_BUNDLE GREATER 1)
